packages/jaeger-ui/src/components/SearchTracePage/SearchForm.jsx

```diff
@@ -276,9 +276,11 @@ export class SearchFormImpl extends React.PureComponent {
     const opsForSvc = (selectedServicePayload && selectedServicePayload.operations) || [];
     const noSelectedService = selectedService === '-' || !selectedService;
     const tz = selectedLookback === 'custom' ? new Date().toTimeString().replace(/^.*?GMT/, 'UTC') : null;
+
     return (
-      <Form layout="vertical" onSubmit={handleSubmit}>
```

Comment:
When the handler was previously on the form itself, what did it mean? Could one submit the form by hitting Enter from any field?

Reply:
Pretty much. Surprisingly though, that still works for me locally even with the handler moved to the submit button.